virtual_rainforest/animals/dummy_animal_module.py

```diff
@@ -0,0 +1,93 @@
+"""The 'dummy_animal_module.
+
+This file provides toy animal module functionality as well as self-contained
+dummy versions of the abiotic, soil, and plant modules that are required for
+setting up and testing the early stages of the animal module.
+"""
+
+# experimental file for figuring out how to make an animal module
+
+# to do
+# - rework dispersal
+# - send portion of dead to carcass pool
+
+# current simplifications
+# - only herbivory (want: carnivory and omnivory)
+# - only endothermy (want: ectothermy)
+# - only iteroparity (want: semelparity)
+# - no development
+
+# notes to self
+# assume each grid = 1 km2
+# assume each tick = 1 day (28800s)
+# damuth ~ 4.23*mass**(-3/4) indiv / km2
+# waste_energy pool likely unnecessary
+#   better to excrete directly to external pools
+# only elephants disperse atm
+
+
+class Plant:
+    """This is a dummy class of plant cohorts for testing the animal module."""
+
+    def __init__(self, name: str, mass: float, position: int) -> None:
+        """The constructor for Plant class."""
+        self.name = name
+        """The functional-type name of the plant cohort."""
+        self.mass = mass
+        """The mass of the plant cohort [g]."""
```

Comment:
Probably it will be a good idea to use a more robust approach for units, using Pint as done in the `soil` model: https://github.com/ImperialCollegeLondon/virtual_rainforest/blob/1df932dec85044aa562757ee8a01d1e560e95f63/virtual_rainforest/soil/model.py#L90

Reply:
I've always worked in grams so its just force of habit. I'm happy to slip over to SI. 

We had a chat about Pint and unit tracking this morning. I think the battle plan is for me to leave it at the moment and then for us to sort out an overall strategy for using Pint in the project and then implement simultaneously. Does that sound reasonable?

Reply:
Sounds reasonable to me!